app/src/main/res/layout/activity_main.xml

```diff
@@ -56,12 +58,13 @@
         android:id="@+id/rcvdImages"
         android:layout_width="@dimen/margin_178dp"
         android:layout_height="@dimen/margin_205dp"
-        android:layout_marginBottom="@dimen/margin_8dp"
+        android:layout_marginStart="8dp"
```

Comment:
Use `@dimen/margin_8dp` instead

Reply:
Okay, please review the new activity_main